examples/temp_linux.py

```diff
@@ -83,7 +83,7 @@ def echo_cmd(dev):
                            format(sub_cmd_in, sub_cmd_out))
 
 def die_temp(dev):
-    dev.cmd(MRPC_DIETEMP, struct.pack("<L", MRPC_DIETEMP_SET_MEAS))
+    #dev.cmd(MRPC_DIETEMP, struct.pack("<L", MRPC_DIETEMP_SET_MEAS))
```

Comment:
This is example code. It really should be clean and clear and having commented out bits is very confusing. Just delete the code if it shouldn't be done that way anymore.

Reply:
Agreed.  I will commit the new change.